beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -46,18 +47,62 @@ func (s *Server) GetLightClientBootstrap(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
-	bootstrap, err := createLightClientBootstrap(ctx, state)
-	if err != nil {
-		httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+	switch blk.Version() {
+	case version.Phase0:
+		httputil.HandleError(w, "light client bootstrap is not supported for phase0", http.StatusNotImplemented)
 		return
-	}
+	case version.Altair:
+		bootstrap, err := createLightClientBootstrap(ctx, state)
+		if err != nil {
+			httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+			return
+		}
 
-	response := &structs.LightClientBootstrapResponse{
-		Version: version.String(blk.Version()),
-		Data:    bootstrap,
-	}
+		response := &structs.LightClientBootstrapResponse{
+			Version: version.String(blk.Version()),
+			Data:    bootstrap,
+		}
+		w.Header().Set(api.VersionHeader, version.String(version.Altair))
 
-	httputil.WriteJson(w, response)
+		httputil.WriteJson(w, response)
+		return
+	case version.Bellatrix:
```

Comment:
For Bellatrix we should return the same thing as for Altair. Just assume there was no change in light client functionality between these 2 forks